views/main.erb

```diff
@@ -10,18 +10,48 @@
   <!-- jquery for status checks -->
   <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
   <script>
-  $(document).ready(function(){
-    $.ajax( 
+  function getSeconds() {
+    // Get the number of seconds since Jan 1, 1970 in UTC
+    return Math.floor(new Date().getTime() / 1000,0); // conversion from ms to s
+  }
+  function periodicTesing() {
+    $.ajax(
       '/check', {
         success: function(data,stat,xhr) {
           // alert("Success " + stat)
         },
         error: function(data,stat,xhr) {
-          $('div.container').first().prepend(data.responseText) 
+          if ($('div.container .checks').length==0) {
+            $('div.container').first().prepend(data.responseText);
+          }
         }
       }
-    )
-  })
+    );
+    $.ajax({
+      url: "/time",
+      success: function(data) {
+        var time_difference = Math.abs(getSeconds() - data);
+        if (time_difference >= 10) {
+          if ($('div.container .timecheck').length==0) {
+            $('div.container').first().prepend(
+              "<div class='timecheck'>\
+                <div class='checkfail'>\
+                  <h4><em>Times do not match.</em></h4>\
+                  <p>Mismatch of " + time_difference + " seconds.</p>\
+                  <p>Please compare local UTC with server UTC.</p>\
```

Comment:
in other words, I'd like to tell them explicitly how to do the fix  (that ^^ command should work _great_ )